pages/common/find.md

```diff
@@ -1,36 +1,29 @@
 # find
 
 > Find files or directories under the given directory tree, recursively.
-> More information: <https://manned.org/find>.
+> See also: `xargs`.
+> More information: <https://manned.org/man/freebsd-13.0/find.1>.
 
-- Find files by extension:
+- Search files by a specific case sensitive glob pattern/case insensitive glob pattern/case sensitive regular expression/case insensitive regular expression:
```

Comment:
Sounds a little hard to understand

Reply:
I think we can limit allowed placeholder count in code examples by linter.

Reply:
1. Another problem exists here: there are two pages: in `linux/` and in `common/`. I can move the page from `common/` to `osx/`. But this command is also [available](http://gnuwin32.sourceforge.net/packages/findutils.htm) in Windows. I can also write `find.md` in `windows/` but there are too many page duplicates. And it hard to support them all. Another decision is to leave just `common/find.md` with the most used options but then we can't always use long options because `common/` pages prefer the most portable options. Personally I keen on short options for fast typing but long ones for memorizing because it's hard to remember option case.
1. There is another question. MacOS doesn's provide GNU find by default. Should we always prefer preinstalled commands on the latest OS version?
1. And the third question. If some command is not preinstalled by default should we describe it at all for this OS? For instance GNU find is not installed by default on Windows.

Reply:
does
```suggestion
- Search files by a specific search pattern:
```
not work here?

Reply:
Sure it's simpler but the user will not understand what to use of the listed alternatives: `{{name|iname|regex|iregex}}`.